app/problem/views.py

```diff
@@ -0,0 +1,29 @@
+""" Endpoint for create problem"""
+from typing import Tuple
+from flask import Blueprint, jsonify, request, Response
+from app.problem.file_creator import FileCreator
+
+problem = Blueprint("problem", __name__)
+
+
+@problem.route("/problem", methods=["POST"])
+def new_problem() -> Tuple[Response, int]:
+    """Endpoint for creating and submitting a new problem to the judge.
+
+    Returns:
+        A JSON response with a success message and a 200 status code if
+        the problem is created successfully.
+    """
+
+    data = request.json
+
+    problem_creator = FileCreator(
+        data.get("problem_id"),
+        data.get("input"),
+        data.get("output"),
+        data.get("points"),
```

Comment:
Esto de los points no se que tanto nos interesa en este momento, deberias comentarlo con David

Reply:
Es que en el ejemplo que me pasó David para el .yml dice points, pero no se de donde salen. Esta es la estructura:

archive: EDM01P01.zip
test_cases:
- {in: EDM01P01.1.in, out: EDM01P01.1.out, points: 5}
- {in: EDM01P01.2.in, out: EDM01P01.2.out, points: 20}
- {in: EDM01P01.3.in, out: EDM01P01.3.out, points: 75}

Reply:
Listo, ya cambie el nombre del archivo y tenías razón jaja me faltaba agregarle algo para los test_cases, resuelto en https://github.com/MinervaLMS/judge-api/commit/df8f7f72b626d4317e9316dfe27208ed32e377f5